### PR TITLE
Added timeline component and wdio test

### LINKED ISSUE #115: Create component, examples and automated test cases for Timeline graph.
# Feature Request

## Description

As an Engineer, I would like to build a Timeline graph component and examples so that we can write automated test cases for the Timeline graph examples.

## Definition of done
We'll know we're done when we complete writing WDIO test cases for the Timeline graph examples.

## Additional Information

Examples needed
1.  Simple​
2.  Axis without tick label

### PR DESCRIPTION
**Pull request by sdadn**
### Summary

Closes #115 

This PR adds a React component for the timeline graph. It also adds a wdio test for a basic timeline graph. This PR also adds a new script in the root `package.json` file for easy recompiling and executing of tests (before these two commands had to be executed separately).

Due to a bug, the criticality example could not be added. This issue has moved to #134

### Deployment Link

https://terra-graphs-deployed-pr-#.herokuapp.com/

### Additional Details

Thank you for contributing to Terra.
@cerner/terra
@cerner/carbon

### CODE REVIEW

File: packages/terra-graphs-docs/src/terra-dev-site/doc/example/Graphs/Timeline/BasicTimeline.jsx [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+import Timeline from '@cerner/terra-graphs/lib/components/Timeline/Timeline';
+import utils from '@cerner/carbon-graphs/lib/js/helpers/utils';
+import '@cerner/terra-graphs-docs/lib/Css/ExampleGraphContainer.module.scss';
+import getTimelineConfig from '@cerner/terra-graphs-docs/lib/example-datasets/graphConfigObjects/Timeline/timelineConfigObject';
+import data from '@cerner/terra-graphs-docs/lib/example-datasets/dataObjects/Timeline/defaultDataset.js';
+
+/*
+Please refer to the documentation below to see the graphConfig and data objects
+*/
+const graphConfig = utils.deepClone(getTimelineConfig('#simpleLinegraph'));
```
Comment (AshishMotanamGurunadham) [inline packages/terra-graphs-docs/src/terra-dev-site/doc/example/Graphs/Timeline/BasicTimeline.jsx:11]: The id should be something related to the timeline graph, like **SimpleTimelineGraph**.

File: packages/terra-graphs-docs/src/terra-dev-site/test/graphs/Timeline/BasicTimeline.test.jsx [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+import Timeline from '@cerner/terra-graphs/lib/components/Timeline/Timeline';
+import utils from '@cerner/carbon-graphs/lib/js/helpers/utils';
+import '@cerner/terra-graphs-docs/lib/Css/ExampleGraphContainer.module.scss';
+import getTimelineConfig from '@cerner/terra-graphs-docs/lib/example-datasets/graphConfigObjects/Timeline/timelineConfigObject';
+import data from '@cerner/terra-graphs-docs/lib/example-datasets/dataObjects/Timeline/defaultDataset.js';
+
+/*
+Please refer to the documentation below to see the graphConfig and data objects
+*/
+const graphConfig = utils.deepClone(getTimelineConfig('#simpleLinegraph'));
```
Comment (AshishMotanamGurunadham) [inline packages/terra-graphs-docs/src/terra-dev-site/test/graphs/Timeline/BasicTimeline.test.jsx:11]: Please change id here.

File: packages/terra-graphs/src/components/Timeline/Timeline.jsx [hunk 3]
```diff
@@ -0,0 +1,67 @@
+/* eslint import/no-unresolved: off */
+import React from 'react';
+import PropTypes from 'prop-types';
+import Carbon from '@cerner/carbon-graphs/lib/js/carbon';
+import '../Graph.module.scss';
+// import './Timeline.module.scss';
```
Comment (AshishMotanamGurunadham) [inline packages/terra-graphs/src/components/Timeline/Timeline.jsx:6]: Don't we need this scss?

File: packages/terra-graphs-docs/src/terra-dev-site/doc/Graphs/Timeline/Simple.1.doc.mdx [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+## Getting Started
+
+- Install with [npmjs](https://www.npmjs.com):
+  - `npm install terra-graphs`
+
+## Usage
+
+```jsx
+import React from 'react';
+import Timeline from '@cerner/terra-graphs/lib/components/Line/Timeline';
```
Comment (AshishMotanamGurunadham) [inline packages/terra-graphs-docs/src/terra-dev-site/doc/Graphs/Timeline/Simple.1.doc.mdx:20]: Sohuld be corrected to 
**import Timeline from '@cerner/terra-graphs/lib/components/Timeline/Timeline';**

File: packages/terra-graphs-docs/src/terra-dev-site/doc/example/Graphs/Timeline/BasicTimeline.jsx (hunk 1 above)
Comment (sdadn) [reply packages/terra-graphs-docs/src/terra-dev-site/doc/example/Graphs/Timeline/BasicTimeline.jsx]: updated here: b1898007ef72fda908cd5275ee447282f5906a56

File: packages/terra-graphs-docs/src/terra-dev-site/test/graphs/Timeline/BasicTimeline.test.jsx (hunk 2 above)
Comment (sdadn) [reply packages/terra-graphs-docs/src/terra-dev-site/test/graphs/Timeline/BasicTimeline.test.jsx]: updated here: b1898007ef72fda908cd5275ee447282f5906a56

File: packages/terra-graphs/src/components/Timeline/Timeline.jsx (hunk 3 above)
Comment (sdadn) [reply packages/terra-graphs/src/components/Timeline/Timeline.jsx]: removed here: a5aed21a6d74dc81b9b61192b5dddcadc455a452

File: packages/terra-graphs-docs/src/terra-dev-site/doc/Graphs/Timeline/Simple.1.doc.mdx (hunk 4 above)
Comment (sdadn) [reply packages/terra-graphs-docs/src/terra-dev-site/doc/Graphs/Timeline/Simple.1.doc.mdx]: fixed here: a5aed21a6d74dc81b9b61192b5dddcadc455a452

File: packages/terra-graphs-docs/src/example-datasets/dataObjects/Timeline/defaultDataset.js [hunk 5]
```diff
@@ -0,0 +1,43 @@
+import Carbon from '@cerner/carbon-graphs/lib/js/carbon';
+import { loadTimelinePopup } from '@cerner/terra-graphs-docs/lib/example-datasets/popup';
+
+const bobRossIpsum = "Just use the old one inch brush. It's so important to do something every day that will make you happy. "
```
Comment (tbiethman) [inline packages/terra-graphs-docs/src/example-datasets/dataObjects/Timeline/defaultDataset.js:4]: +1 for happy trees

**Review by tbiethman — APPROVED** [review]

**Review by mjhenkes — APPROVED** [review]

**Review by benbcai — APPROVED** [review]

**Review by AshishMotanamGurunadham — APPROVED** [review]

**Pull request merged**